### ISSUE TITLE
Brizy creates many files, consequently many INODES per site

### ISSUE THREAD

**Issue by edugsdf** [ThemeFuse/Brizy#1599]
Brizy creates many files, consequently many INODES per site. When you have several websites the impact of INODES on the servers is huge.
And all hosting has this limit of INODES
![2019 12 25-Brizi-INODEs](https://user-images.githubusercontent.com/8173351/71447351-f8c4ec00-270b-11ea-99c6-c7db6cbf25bb.png)

**Comment by rfmurphy81** [comment]
Yes, please prioritize this as most shared web hosts have limits on the number of inodes available and I've noticed that the number of files that Brizy uses is higher than expected. This would be a welcome improvement/optimization for 2020!

**Comment by bagrinsergiu** [comment]
Hi,

We have 2 kits and ~250 pages for layouts. All of them are json files. I need to think how we can solve this issue. But it is very strange is just a 20kb of json 

Regards 
Sergiu

**Comment by franckadil** [comment]
Hey team, what about //wp-content/uploads/backup/ is it created by Brizy ?
